depends/packages/qt.mk

```diff
@@ -1,53 +1,114 @@
 PACKAGE=qt
-$(package)_version=5.2.1
-$(package)_download_path=http://download.qt-project.org/official_releases/qt/5.2/$($(package)_version)/single
-$(package)_file_name=$(package)-everywhere-opensource-src-$($(package)_version).tar.gz
-$(package)_sha256_hash=84e924181d4ad6db00239d87250cc89868484a14841f77fb85ab1f1dbdcd7da1
+$(package)_version=5.4.2
+$(package)_download_path=http://download.qt.io/official_releases/qt/5.4/$($(package)_version)/submodules
```

Comment:
I'd was about to say belt and suspenders to use https, especially as the server supports it. Being slightly paranoid hasn't hurt us in the past.
But looks like using https makes no sense, it redirects you to a http mirror:
```
--2015-06-24 15:40:51--  https://download.qt.io/official_releases/qt/5.4/5.4.2/submodules/qtx11extras-opensource-src-5.4.2.tar.xz
Resolving download.qt.io (download.qt.io)... 77.86.229.90
Connecting to download.qt.io (download.qt.io)|77.86.229.90|:443... connected.
HTTP request sent, awaiting response... 302 Found
Location: http://ftp2.nluug.nl/languages/qt/archive/qt/5.4/5.4.2/submodules/qtx11extras-opensource-src-5.4.2.tar.xz [following]
--2015-06-24 15:40:52--  http://ftp2.nluug.nl/languages/qt/archive/qt/5.4/5.4.2/submodules/qtx11extras-opensource-src-5.4.2.tar.xz
Resolving ftp2.nluug.nl (ftp2.nluug.nl)... 192.87.102.43, 2001:610:1:80aa:192:87:102:43
Connecting to ftp2.nluug.nl (ftp2.nluug.nl)|192.87.102.43|:80... connected.
HTTP request sent, awaiting response... 200 OK
```